utils/helpers.js

```diff
@@ -14,6 +14,13 @@ export const addWorkoutDays = (number) => {
   return store;
 };
 
+export const fetcher = (...args) =>
+  fetch(...args).then((response) => response.json());
+
+export const usePlan = (id) => useSWR(`/api/plans/${id}`, fetcher);
+
+export const useAllPlans = () => useSWR(`/api/plans`, fetcher);
```

Comment:
You can consider to configure the `fetcher` globally in the `_app.js` instead so that you don't need to pass it every time. Please refer to the handout of the React Data Fetching session how to do that.

Reply:
Thanks!
I'll look into it.